### PR TITLE
Add transfer vote recording and verbosity in count endpoint 

### PR DESCRIPTION
**Pull request by mpancia**
@DSASanFrancisco/portal-members

### CODE REVIEW

File: membership/web/util.py [hunk 1]
```diff
@@ -15,7 +20,7 @@ def __init__(self, err: str) -> None:
             json.dumps(payload), status=400, mimetype='application/json')
 
 
-def new_alchemy_encoder(fields_to_expand: List[str]=[]):
+def new_alchemy_encoder(fields_to_expand: List[str] = []):
```
Comment (jeffmay) [inline membership/web/util.py]: Default args in Python don't have spaces around them.

File: membership/web/elections.py [hunk 2]
```diff
@@ -188,7 +189,15 @@ def election_count(requester: Member, session: Session):
...
-    return jsonify({'winners': winners})
+    round_information = {}
+    for round_number, round in enumerate(stv.previous_rounds):
+        candidate_information = {}
+        for cid, vote_info in round.items():
+            candidate_name = session.query(Candidate).get(cid).member.name
+            candidate_information[candidate_name] = vote_info
+        round_information[round_number + 1] = candidate_information
+    return custom_jsonify(data={'winners': winners, 'round_information': round_information},
+                          encoder=CustomEncoder)
```
Comment (jeffmay) [inline membership/web/elections.py]: How are we storing this information after an election is counted?

File: membership/web/util.py [hunk 3]
```diff
@@ -48,3 +53,23 @@ def default(self, obj):
             return json.JSONEncoder.default(self, obj)
 
     return AlchemyEncoder
+
+
+class CustomEncoder(JSONEncoder):
+    """ Custom encoder class converts Decimals to strings and datetime objects into ISO
+    ISO formatted strings. """
```
Comment (jeffmay) [inline membership/web/util.py]: `ISO ISO formatted strings` I didn't know this ISO had their own ISO format :trollface:

Comment (mpancia) [inline membership/web/util.py]: Copied from @jesse-sempre :P

File: membership/web/util.py (hunk 1 above)
Comment (mpancia) [inline membership/web/util.py]: Blaming auto-pycharm formatting. Will fix.

File: membership/web/elections.py (hunk 2 above)
Comment (mpancia) [inline membership/web/elections.py]: It's not stored -- the election itself retains all of the information that it needs to `hold_election`, but that's not persisted. Does it need to be?

**Comment by jessebutterfield** [comment]
@jeffmay I wanted to standardize the format a little before we created a results table since I wasn't sure what we'd want in there. But yeah, eventually we'll definitely want one.

**Comment by jeffmay** [comment]
As I thought about it.. the result is pretty much derrived data. The only issue is if there is a random tie break, then we would want to store who won the toss-up.

**Comment by jessebutterfield** [comment]
I actually opened a ticket on trello about this, I think we should seed the random generator based on the election id so the election results are always reproducible.

**Pull request merged**